hibernate-core/src/main/java/org/hibernate/dialect/Dialect.java

```diff
@@ -1980,6 +1980,46 @@ public final String quote(String name) {
 	}
 
 	/**
+	 * Does this dialect support catalog creation?
+	 *
+	 * @return True if the dialect supports catalog creation; false otherwise.
+	 */
+	public boolean canCreateCatalog() {
+		return false;
+	}
+
+	/**
+	 * Get the SQL command used to create the named catalog
+	 *
+	 * @param catalogName The name of the catalog to be created.
+	 *
+	 * @return The creation command
+	 */
+	public String getCreateCatalogCommand(String catalogName) {
```

Comment:
Also, as we discussed on IRC we might want this to allow returning a String array